Gemfile

```diff
@@ -80,16 +80,13 @@ gem 'email_prefixer'
 gem 'puma', '~> 3.2'
 
 # Attach comments to Active Record queries
-gem 'marginalia'
+gem 'marginalia', github: 'basecamp/marginalia'
 
```

Comment:
Reference https://github.com/basecamp/marginalia/issues/58

New version with latest changes is not yet released